pkg/internal/tests/cross-tests/input_cross_test.go

```diff
@@ -103,93 +103,6 @@ func TestInputsEqualObjectBasic(t *testing.T) {
 	}
 }
 
-func TestInputsConfigModeEqual(t *testing.T) {
```

Comment:
This test was moved to provider_test.go.

Reply:
Not clear why this was moved and not the other tests. Can you please clarify?

Reply:
The test was moved, and re-written (from tftypes.Value) to (cty.Value). Other tests were not moved only due to the cost of moving them (rewriting values).

Reply:
I added more clarity in https://github.com/pulumi/pulumi-terraform-bridge/pull/2508.